harness/propertyHelper.js

```diff
@@ -268,3 +269,25 @@ function verifyNotConfigurable(obj, name) {
     throw new Test262Error("Expected obj[" + String(name) + "] NOT to be configurable, but was.");
   }
 }
+
+function verifyBuiltinProperty(obj, name, desc, options) {
+  assert(
+    arguments.length > 2,
+    'verifyBuiltinProperty should receive at least 3 arguments: obj, name, and descriptor'
+  );
+
+  if (desc && (true === desc.writable) || (true === desc.configurable)) {
+    const LOCKED_DOWN = ($262 && (typeof $262.isLockedDown === "function")) ? $262.isLockedDown() : false;
```

Comment:
Make sure to document `isLockedDown` in [INTERPRETING.md](https://github.com/tc39/test262/blob/main/INTERPRETING.md).

Reply:
Will do.